modules/local/get_software_versions.nf

```diff
@@ -22,9 +24,9 @@ process GET_SOFTWARE_VERSIONS {
 
     input:
     path versions
-    
+
     output:
-    path "software_versions.csv"     , emit: csv
+    path "software_versions.tsv"     , emit: csv
```

Comment:
I guess this could be named tsv in the future at some point :D

Reply:
ok, just updated!